DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js

```diff
@@ -157,6 +181,15 @@ module.exports = function(ngModule) {
         vm.addingWorkSite = true;
         vm.setActiveTab(1);
       }
+
+      vm.employeeColumnDefs[8] = {
+        targets:1,
+        createdCell: function (td, cellData, rowData, row) {
+          if ($scope.validate('workSites[' + index + '].employees[' + row + ']')) {
+            $(td).prepend("<span class='usa-input-error-message' role='alert' tabindex='0'>Please review this employee and correct any errors</span>")
+          }
+        }
+      };
```

Comment:
Similar question as above re: adding column definitions, except in this case it's not using `push`.  Looking at the `tableConfig.js`, it looks like the last index is 7, so assigning this to index 8 is effectively the same as using `push`.  The problem is if `tableConfig.js` is ever updated to add or remove some elements, this code will no longer work as expected.